eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -0,0 +1,160 @@
+function Get-java-OnboardedDocsMsPackages($DocRepoLocation) {
+    $packageOnboardingFiles = "$DocRepoLocation/package.json"
+  
+    $onboardingSpec = ConvertFrom-Json (Get-Content $packageOnboardingFiles -Raw)
+    $allPackages = @{}
+    foreach ($spec in $onboardingSpec) {
+      $spec.packages | ForEach-Object {$allPackages[$_.packageArtifactId] = $null}
+    }
+    return $allPackages
```

Comment:
I'm not too familiar with how these functions get imported. Where is this object used and could it just be an array instead?

Reply:
It is used to dedup